kratos/mpi/tests/test_mpi_model_part.py

```diff
@@ -1,31 +1,12 @@
 import KratosMultiphysics
 import KratosMultiphysics.KratosUnittest as KratosUnittest
+from KratosMultiphysics.testing.utilities import ReadDistributedModelPart
 
 import os
 
 def GetFilePath(fileName):
     return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
 
-def ReadDistributedModelPart(model_part, mdpa_file_name):

```

Comment:
Ping @KratosMultiphysics/altair this might break some of your tests if I recall correctly, be careful

Reply:
@KratosMultiphysics/altair if this breaks your tests then please use the same fct I use in this PR (it does the same)

Reply:
I think most tests should be fine. Thanks for pinging! ;)